### ISSUE TITLE
Docker build fails with USE_CUSTOM_SDKS=true

### ISSUE THREAD

**Issue by williamkray** [vector-im/element-web#16305]
VERSION: v1.7.18

Building using the Dockerfile works fine when leaving `USE_CUSTOM_SDKS=false`. However, setting to `USE_CUSTOM_SDKS=true` even if the default SDK endpoints are left intact results in the following vague build error:

```
...
<s> [webpack.Progress] 69% building 1073/1075 modules 2 active css /src/node_modules/css-loader/dist/cjs.js??ref--6-1!/src/node_modules/postcss-loader/src/index.js??postcss!/src/react-sdk/res/themes/dark/css/dark.scss 0
<s> [webpack.Progress] 69% building 1074/1075 modules 1 active /src/node_modules/mini-css-extract-plugin/dist/loader.js!/src/node_modules/css-loader/dist/cjs.js??ref--6-1!/src/node_modules/postcss-loader/src/index.js??postcss!/src/react-sdk/res/themes/light/css/light.scss
<s> [webpack.Progress] 70% building 1075/1075 modules 0 active
<s> [webpack.Progress] 69% building 1075/1076 modules 1 active css /src/node_modules/css-loader/dist/cjs.js??ref--6-1!/src/node_modules/postcss-loader/src/index.js??postcss!/src/react-sdk/res/themes/light/css/light.scss 0
<s> [webpack.Progress] 70% building 1076/1076 modules 0 active
<s> [webpack.Progress] 70% building 1076/1076 modules 0 active
<s> [webpack.Progress] 70% building 1076/1076 modules 0 active
<s> [webpack.Progress] 70% building 1076/1076 modules 0 active
<s> [webpack.Progress] 70% building 1076/1076 modules 0 active
<s> [webpack.Progress] 70% building 1076/1076 modules 0 active
<s> [webpack.Progress] 70% building 1076/1076 modules 0 active
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
The command '/bin/sh -c yarn build' returned a non-zero code: 1
```

This means I am unable to merge in my patches to the latest stable release.

**Comment by williamkray** [comment]
i have the same behavior on v1.7.17 as well. last stable working build i had was 1.7.16, and i built a container from the develop branches around January 15th.

**Comment by jryans** [comment]
I believe this is the same as https://github.com/vector-im/element-web/issues/16119. https://github.com/vector-im/element-web/pull/16192 should be the fix, which only exists in 1.7.19-rc.1 so far. Can you check if this helps?

**Comment by williamkray** [comment]
looks like pulling in that commit allowed the build to complete! thanks!
